tests/AbstractTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ *
+ */
+
+class AbstractTest extends \PHPUnit\Framework\TestCase
+{
+    public static function setUpBeforeClass() : void
```

Comment:
Removing these methods will probably let us test the project under earlier versions of PHP.

Reply:
Done